index.dd

```diff
@@ -640,7 +640,7 @@ import core.stdc.stdlib;
 void livingDangerously()
 {
     // Access to C's malloc and free primitives
-    auto buf = malloc(1024 * 1024);
+    auto buf = malloc(float.sizeof * 1024 * 1024);
```

Comment:
This wasn't a bug. The extra memory you're allocating here isn't being used below.